frontend/README.md

```
@@ -18,10 +18,22 @@ Based on:
-2. Create local env file: `cp .env.example .env.local`
-3. Run dev server: `pnpm dev`
-4. Run tests: `pnpm test`
+## Requirements
+
+- Node.js 24 LTS
+- pnpm 11.5.2 via Corepack
+
+`package.json` and `netlify.toml` are the source of truth for the frontend
+Node.js and pnpm versions.
+
+## Setup
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)